django/db/backends/sqlite3/base.py

```diff
@@ -169,6 +169,27 @@ def get_new_connection(self, conn_params):
         conn.create_function("regexp", 2, _sqlite_regexp)
         conn.create_function("django_format_dtdelta", 3, _sqlite_format_dtdelta)
         conn.create_function("django_power", 2, _sqlite_power)
+        conn.create_function('LPAD', 3, _sqlite_lpad)
+        conn.create_function('RPAD', 3, _sqlite_rpad)
+        conn.create_function('ACOS', 1, math.acos)
+        conn.create_function('ASIN', 1, math.asin)
+        conn.create_function('ATAN', 1, math.atan)
+        conn.create_function('ATAN2', 2, math.atan2)
+        conn.create_function('CEILING', 1, math.ceil)
+        conn.create_function('COS', 1, math.cos)
+        conn.create_function('COT', 1, lambda x: 1 / math.tan(x))
+        conn.create_function('DEGREES', 1, math.degrees)
+        conn.create_function('EXP', 1, math.exp)
+        conn.create_function('FLOOR', 1, math.floor)
+        conn.create_function('LN', 1, lambda x: math.log(x))
+        conn.create_function('LOG', 2, lambda x, y: math.log(y, x))
+        conn.create_function('MOD', 2, lambda x, y: math.fmod(x, y))
```

Comment:
The lambda wrapper is not need here.

Reply:
I have updated the line
conn.create_function('LN', 1, lambda x: math.log(x))
to
conn.create_function('LN', 1, math.log)

I think the functions with two arguments should keep the lambda wrapper. Correct me if I was wrong. Thanks!

Reply:
I was referring to `lambda x, y: math.fmod(x, y)` where the arguments are passed in the same order.

Reply:
I see. I have updated the conn.create_function('MOD', ...) and conn.create_function('ATAN2', ...) since atan2 also has arguments in the same order.